packages/gridstack/src/components/DashboardBar/DashboardBar.vue

```diff
@@ -2,25 +2,34 @@
   <div class="dashboard-bar">
     <h1>Dashboard</h1>
     <div class="right-aligned">
+      <SelectInput
+        :options="dashboardConfigurationOptionsMap"
+        :label="'Dashboard-Konfiguration'"
+        v-if="!editMode"
+        v-model:model-value="selectedDashboardId"
```

Comment:
If the corresponding prop is `modelValue` you can use the v-model shorthand
```suggestion
        v-model="selectedDashboardId"
```
The position of the v-if and the v-model should be shifted to the top. Seems that the linter isn't working right.